chaoslib/litmus/pod_failure_by_litmus.yml

```diff
@@ -7,22 +7,7 @@
     chaos_iterations: 1
   when: "chaos_iterations == '0'" 
 
-- name: Kill application pods randomly for the specified duration
-  shell: |
-    kubectl get pods -l {{ a_label }} -n {{ a_ns }} --no-headers -o custom-columns=:metadata.name | shuf -n 1 | xargs kubectl delete pod -n {{ a_ns }} --force --grace-period=0 --wait=false
-    sleep {{ c_interval }}
-  args:
-    executable: /bin/bash
-  register: result
+- name: Kill random pod
+  include: kill_random_pod.yml
```

Comment:
Just thinking out loud: Should we have a chaoslib/litmus/pod-delete/<util> to make it more specific and segregated?  Also, how about a naming convention to distinguish between the actual lib_utils & auxiliary ones it makes use of - like the `kill_random_pod.yml` ? One of the needs of doing this is the fact that everything in ansible is a yml - and it may `look` too repetitive if names and directories don't paint a good picture. 

 Looking at something like the below: 

chaoslib
|_litmus
   |_ pod-delete
      |_ pod_failure_by_litmus.yml
          utils
             |_ kill_random_pod.yml

Reply:
Proabably a good time to decide this and then the same can be adopted for other experiments going forward.

Reply:
Wouldn't it be better to keep `utils` folder under /chaoslib/litmus itself? we can dump the reusable utils in that folder which is specific to `litmus`.
Also, the word `by_litmus` seems redundant to me, as it is already inside `litmus` folder

Reply:
Agree on the `by_litmus` redundancy. There is a `utils/common` already which has been created to hold common tasks which can be reused both by main experiment files & chaoslib. The idea of dedicated `utils` under respective libs is to keep tightly coupled fault-injection logic in one place - this might manifest itself when doing more involved cases that are, say, tool or platform dependent. Having said that, it might even be sufficient to create the experiment specific folders inside chaoslib/<lib>..for to this purpose.

Reply:
Any further thoughts?

Reply:
This could be a separate PR, if necessary.